firefox-ios/Client/FennecTesting.xcconfig

```diff
@@ -0,0 +1,9 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+// Configuration settings file format documentation can be found at:
+// https://help.apple.com/xcode/#/dev745c5c974
+
+SWIFT_ACTIVE_COMPILATION_CONDITIONS = TESTING;
+OTHER_SWIFT_FLAGS = $(OTHER_SWIFT_FLAGS_common) -DMOZ_CHANNEL_FENNEC -DTESTING
```

Comment:
We should also import the `Fennec.xcconfig` file into this one since they are dependent.
```
include "Fennec.xconfig"
```
 and i would remove -DTESTING as we concluded the last time those are the same as setting active compilation conditions

Reply:
I was facing issues with importing `include "Fennec.xcconfig"` since it could not be found. This may be due to not having a selected `Target Membership`. It seems to be fine locally and in bitrise without it, so didn't think it was necessary to add it + didn't want to make additional changes unless needed.

Removed that extra flag!

Reply:
Ok perfect thanks Cindy. I tought without could break tests. Perfect!! PR Approved

Reply:
great! thanks for the thorough review @FilippoZazzeroni !